percy-csharp-selenium/Percy.cs

```diff
@@ -15,63 +15,82 @@ namespace percy_csharp_selenium
    */
     public class Percy
     {
-
-        // We'll expect this file to exist at the root of our classpath, as a resource.
-        private static readonly String AGENTJS_FILE = "percy-agent.js";
-
         // Selenium WebDriver we'll use for accessing the web pages to snapshot.
         private IWebDriver driver;
 
-        // The JavaScript contained in percy-agent.js
-        private String percyAgentJs;
+        // The JavaScript contained in dom.js
+        private String domJs = "";
+
+        // Maybe get the CLI server address if not Set the CLI server address 
+        //could be moved to percy-csharp-selenium Environment
+        private String PERCY_SERVER_ADDRESS = System.Environment.GetEnvironmentVariable("PERCY_SERVER_ADDRESS")!=null?
+                System.Environment.GetEnvironmentVariable("PERCY_SERVER_ADDRESS") : "http://localhost:5338";
 
         // Environment information like the programming language, browser, & SDK versions
         private Environment env;
```

Comment:
We need to have an equivalent of this.
```
 // Determine if we're debug logging
private boolean PERCY_DEBUG = System.getenv("PERCY_LOGLEVEL").equals("debug");
```

` // for logging
private String LABEL = "[\u001b[35m" + (PERCY_DEBUG ? "percy:java" : "percy") + "\u001b[39m]";`